tripy/tripy/types.py

```diff
@@ -0,0 +1,40 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2024-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Special type annotations used in Tripy.
+"""
+
+import numbers
+import sys
+from typing import Union, Sequence
+
+from tripy import export
+
+export.public_api()(sys.modules[__name__])
+
+tensor_literal = export.public_api(
+    document_under="types.rst",
+    autodoc_options=[":no-index:"],
+    module=sys.modules[__name__],
+    symbol="tensor_literal",
+)(Union[numbers.Number, Sequence["tripy.types.tensor_literal"]])
+
+tensor_literal.__doc__ = """Denotes the recursive type annotation for tensor literals.
```

Comment:
This was weird, but it was the only way I could add in a doc that the UX test recognized. Is there another way to add a docstring to something that's not a class or function?

Reply:
You can just add it directly below as a docstring:
```
tensor_literal = ...
"""
Denotes the ...
"""
```

Reply:
That's weird. It seems to work fine in `config.py`